homeassistant/components/switcher_kis/light.py

```diff
@@ -35,16 +35,30 @@ async def async_setup_entry(
     def async_add_light(coordinator: SwitcherDataUpdateCoordinator) -> None:
         """Add light from Switcher device."""
         entities: list[LightEntity] = []
-        if (
-            coordinator.data.device_type.category
-            == DeviceCategory.SINGLE_SHUTTER_DUAL_LIGHT
+        device_type_to_light_count = {
+            DeviceType.RUNNER_S11: 2,
+            DeviceType.RUNNER_S12: 1,
+            DeviceType.LIGHT_SL01: 1,
+            DeviceType.LIGHT_SL01_MINI: 1,
+            DeviceType.LIGHT_SL02: 2,
+            DeviceType.LIGHT_SL02_MINI: 2,
+            DeviceType.LIGHT_SL03: 3,
+        }
```

Comment:
After looking at the full list, the number is identical to the light array length, why can't we use 
`len(coordinator.data.light)` as the number of outputs?

Reply:
It is done.